components/dashboard/src/settings/Preferences.tsx

```diff
@@ -160,38 +160,6 @@ export default function Preferences() {
                                         </ul>
                                     </InfoBox>
                                 )}
-                            </>
-                        )}
-                        {desktopIdeOptions && (
-                            <>
-                                <h3 className="mt-12 flex">
-                                    Desktop Editor
-                                    <PillLabel type="warn" className="font-semibold py-0.5 px-2 self-center">
-                                        Beta
-                                    </PillLabel>
-                                </h3>
-                                <p className="text-base text-gray-500 dark:text-gray-400">
-                                    Optionally, choose the default desktop editor for opening workspaces.
-                                </p>
-                                <div className="my-4 gap-4 flex flex-wrap max-w-2xl">
-                                    {desktopIdeOptions.map(([id, option]) => {
-                                        const selected = defaultDesktopIde === id;
-                                        const onSelect = () => actuallySetDefaultDesktopIde(id);
-                                        if (id === DesktopNoneId) {
-                                            option.logo = isDark ? IDENoneDark : IDENone;
-                                        }
-                                        return renderIdeOption(option, selected, onSelect);
-                                    })}
-                                </div>
-                                {ideOptions.options[defaultDesktopIde]?.notes && (
-                                    <InfoBox className="my-5 max-w-2xl">
-                                        <ul>
-                                            {ideOptions.options[defaultDesktopIde].notes?.map((x, idx) => (
-                                                <li className={idx > 0 ? "mt-2" : ""}>{x}</li>
-                                            ))}
-                                        </ul>
-                                    </InfoBox>
-                                )}
                                 <p className="text-left w-full text-gray-500">
                                     The <strong>JetBrains desktop IDEs</strong> are currently in beta.{" "}
```

Comment:
**question**: Posting one last question here for future reference. Do we still consider JetBrains editor support beta? Maybe this beta notice is something to consider removing in a future PR. Cc @akosyakov @loujaybee

Reply:
Yes, everything is in BETA but VS Code Web.

Reply:
Thanks for clarifying, @iQQBot @akosyakov! 🙏